### DISCUSSION TITLE
WARNING ⚠️ 'Boxes.boxes' is deprecated. Use 'Boxes.data' instead.

### DISCUSSION THREAD

**Discussion by robmarkcole** [obss/sahi#869] [Q&A]
This WARNING is printed on every `get_sliced_prediction` and overwhelming the logs, can it be disabled?

**Comment by Sylus1000** [comment]
See: https://github.com/ultralytics/ultralytics/blob/af49a85cf3e0590e2c79be2f42cc54c3c22c1f62/ultralytics/yolo/engine/results.py#L433
